packages/utils/src/partial_ord.cairo

```diff
@@ -0,0 +1,47 @@
+/// PartialOrd implementation for tuple (u32, felt252).
+/// Required to sort leaf nodes in proof verification.
+pub impl PartialOrdTupleU64Felt252 of PartialOrd<(u64, felt252)> {
```

Comment:
I'd not put this to utils as it's still specific to utreexo

Reply:
ok!